lib/comfortable_mexican_sofa/content/params_parser.rb

```diff
@@ -80,6 +82,7 @@ def self.split_on_commas(tokens)
   # @param [String] args_string
   # @return [Array<String>] tokens
   def self.tokenize(args_string)
+    args_string = args_string.dup if args_string.frozen?
     args_string.tr!("“”‘’", %q(""''))
```

Comment:
cant we just do reassignment instead of conditional duping? `args_string = args_string.tr("“”‘’", %q(""''))`

Reply:
Done